docs/netdata-cloud/signing-in.md

```diff
@@ -0,0 +1,158 @@
+
+![A screenshot of Netdata's web interface](https://user-images.githubusercontent.com/1153921/59644657-b7330300-9122-11e9-9dda-ea784422f3f2.png)
+
+From here, you need to register for a Netdata Cloud account. Click on the `Sign in` button on the top-right corner of the dashboard's view.
+
+![A screenshot of the Sign in button in the Netdata dashboard](https://user-images.githubusercontent.com/1153921/59782688-6252d200-9273-11e9-9975-52be0d6714bf.png)
+
+??? note "Alternative registration routes"
+    While we recommend the `Sign in` button, the Netdata dashboard has one other direct route registering for or signing in to a Netdata Cloud account.
+
+    The text `Please sign in to netdata.cloud to view your nodes!` contains a link to access Netdata Cloud.
+    
+    ![A screenshot of the Netdata Cloud sign in link](https://user-images.githubusercontent.com/1153921/59644958-2f4df880-9124-11e9-946c-bb30c8735e0a.png)
+
+    Two other routes exist, but they are more directly related to accessing the Nodes View. They will, however, require either registration or sign in and thus are valid routes to access Netdata Cloud.
+
+    One route can be found in the `Nodes<sup>Beta</sup>` button the left side of the navigation menu:
+
+    ![A screenshot of a link to the Nodes View in Netdata Cloud](https://user-images.githubusercontent.com/1153921/59644663-c1ed9800-9122-11e9-9ebc-d67e7db229a7.png)
+
+    A second route can be found in the Nodes List—the drop-down menu in the top-left corner of the Netdata dashboard:
+
+     ![A screenshot of a second link to the Nodes View in Netdata Cloud](https://user-images.githubusercontent.com/1153921/59644973-3d9c1480-9124-11e9-9a1d-33c412578a9f.png)
+
+
+??? note "Registration route when using a private registry"
+    If you're using a private registry, clicking the `Sign in` button will display a modal window warning you about the process of migrating away from your private registry and to Netdata Cloud's registry.
+
+    ![A screenshot of the private registry warning modal](https://user-images.githubusercontent.com/1153921/59782901-ca091d00-9273-11e9-9f9a-0cb18f78ca26.png)
+
+    If you agree to use Netdata Cloud over your private registry, and opt-in to let Netdata Cloud receive and store the information described [here](../../registry/README.md#what-data-does-the-registry-store), you should click the `Sign in` button again. If not, click the `Cancel` button to continue using your private registry.
+    
+
+### Choosing your registration or sign in method
+
+After clicking the `Sign in` button, you'll be directed to the Netdata Cloud registration/sign in page. Choose to authorize with your Google account, GitHub account, or email.
+
+!!! attention
+    Be consistent with the sign in method you use, whether GitHub, Google, or email. If you sign in via different methods, the system may accidentally create multiple Netdata Accounts. We are currently working on a fix for this inconsistent behavior.
```

Comment:
"the system will create multiple Netdata Accounts, one for each sign-in method".  We plan to offer multiple authentication methods for the same account in the future.